src/gsy_e/models/strategy/area_agents/two_sided_agent.py

```diff
@@ -15,35 +15,41 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+from gsy_framework.constants_limits import ConstSettings
 from numpy.random import random
+
 from gsy_e.models.strategy.area_agents.one_sided_agent import OneSidedAgent
 from gsy_e.models.strategy.area_agents.two_sided_engine import TwoSidedEngine
-from gsy_framework.constants_limits import ConstSettings
 
 
 class TwoSidedAgent(OneSidedAgent):
+    """Handles order forwarding between two sided markets."""
 
     def __init__(self, *, owner, higher_market, lower_market,
                  min_offer_age=ConstSettings.IAASettings.MIN_OFFER_AGE,
-                 min_bid_age=ConstSettings.IAASettings.MIN_BID_AGE):
-        self.engines = [
-            TwoSidedEngine('High -> Low', higher_market, lower_market, min_offer_age,
-                           min_bid_age, self),
-            TwoSidedEngine('Low -> High', lower_market, higher_market, min_offer_age,
-                           min_bid_age, self),
-        ]
+                 min_bid_age=ConstSettings.IAASettings.MIN_BID_AGE,
+                 do_create_engine=True):
+        if do_create_engine:
```

Comment:
Ummm I don't find this approach clean tbh
In case this is here in order to control what type of engines to create alongside the mro, I suppose we can create a dedicated method that selects the wanted engine, and we can overwrite it later on

Reply:
ok, let me try this.